topics/single-cell/tutorials/EBI-retrieval/tutorial.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: tutorial_hands_on
+subtopic: datamanipulation
+priority: 3
+title: EBI Single Cell Expression Atlas files to AnnData | Creating preprocessed dataset for sc-RNA Filter, Plot, Explore tutorial
+questions:
+- How do I use EBI Single Cell Expression Atlas?
+- How can I reformat and manipulate the downloads to create the right input for downstream analysis?
+objectives:
+- You will retrieve raw data from the EBI Single Cell Expression Atlas.
+- You will manipulate the metadata and matrix files.
+- You will combine the metadata and matrix files into an AnnData object for downstream analysis.
+  
+time_estimation: "15m"
+key_points:
+- The EMBL-EBI Single-cell expression atlas contains high quality datasets.
+- Metadata manipulation is key for generating the correctly formatted resource.
+- To use Scanpy tools, you have to transform your metadata into AnnData object.
+contributions:
+  authorship:
+    - wee-snufkin
+  funding: 
+  - elixir-fair-data
+
+requirements:
+  -
+    type: "internal"
+    topic_name: single-cell
+    tutorials:
+      - scrna-case_alevin
+      - scrna-case_alevin-combine-datasets
+
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: single-cell
+    tutorials:
+        - scrna-case_basic-pipeline
+
+tags:
+  - transcriptomics
```

Comment:
```suggestion
  - data import
```

It's not strictly bulk transcriptomics so hence we've removed these from anything where that's the case